docs/main/user-guides/replicated-storage-user-guide/rs-installation.md

```diff
@@ -272,13 +274,27 @@ echo vm.nr_hugepages = 1024 | sudo tee -a /etc/sysctl.conf
 ```
 
 :::warning
-If you modify the Huge Page configuration of a node, you _MUST_ either restart kubelet or reboot the node.  replicated engine will not deploy correctly if the available Huge Page count as reported by the node's kubelet instance does not satisfy the minimum requirements.
+If you modify the huge page configuration of a node, you _MUST_ either restart kubelet or reboot the node.  Replicated Storage will not deploy correctly if the available huge page count as reported by the node's kubelet instance does not satisfy the minimum requirements.
 :::
 
-### Label Replicated Engine Node Candidates
+#### Label Replicated Storage Node Candidates
 
-All worker nodes which will have replicated engine pods running on them must be labelled with the OpenEBS engine type "replicated engine". This label will be used as a node selector by the Replicated Engine Daemonset, which is deployed as a part of the replicated engine data plane components installation. To add this label to a node, execute:
+All worker nodes which will have Replicated Storage pods running on them must be labelled with the OpenEBS storage type "Replicated Storage". This label will be used as a node selector by the Replicated Storage Daemonset, which is deployed as a part of the Replicated Storage data plane components installation. To add this label to a node, execute:
```

Comment:
1. IO engine pods running on them
2. type "replicated engine" is correct
3. by the IO engine Daemonset